test/lib/CancelOtherScript.sol

```diff
@@ -2,11 +2,24 @@
 pragma solidity 0.8.23;
 
 import "quark-core/src/QuarkWallet.sol";
+import "quark-core/src/QuarkScript.sol";
 
-contract CancelOtherScript {
+contract CancelOtherScript is QuarkScript {
```

Comment:
does this contract actually make any sense anymore? I don't think it really does

you could rename it `Cancel` but it's at least no longer a `CancelOther`, and really it's a `nop` that you could just as easily run to emit the `CancelNonce()` event erroneously...